introduction/examples/thompson/fibonacci.py

```diff
@@ -0,0 +1,48 @@
+def list_fibonacci(n):
+    if isinstance(n, int):
+        if n >= 0:
+            list1 = [0]
+            pre = 0
+            post = 1
+            for i in range(n):
+                tmp = pre + post
+                pre = post
+                post = tmp
+                list1.append(pre)
+            return list1
+        return 0
+    return 0
+
+
+def iter_fibonacii(n):
+    if isinstance(n, int):
+        pre = 0
+        post = 1
+        for i in range(n):
+            tmp = pre + post
```

Comment:
Podobnie jak wyżej. Można zamienić w miejscu (na razie traktuj to jako ciekawostkę)